src/bundle/Resources/public/js/scripts/admin.location.view.js

```diff
@@ -55,13 +59,56 @@
             }
         });
     };
-    const generateLink = (locationId) => window.Routing.generate('_ezpublishLocation', { locationId });
+    const generateLink = (locationId) => global.Routing.generate('_ezpublishLocation', { locationId });
+    const setModalTitle = (title) => {
+        const modalTitleNode = doc.querySelector(`${SELECTOR_BULK_OPERATION_FAILED_MODAL} .modal-header .modal-title`);
 
-    listContainers.forEach(container => {
+        modalTitleNode.innerHTML = title;
+    };
+    const setModalTableBody = (failedItemsData) => {
+        const modal = doc.querySelector(SELECTOR_BULK_OPERATION_FAILED_MODAL);
+        const table = modal.querySelector('table');
+        const tableBody = table.querySelector('tbody');
```

Comment:
Selectors as a tag name are not the best. Please add some classes to this table.

Reply:
Do we want class on every table element?

Reply:
On the elements that are relevant from JS or CSS perspective

Reply:
My question was a bit imprecise, if we want to refer in JS `table` and `tbody` elements and `tbody` is inside `table`, we need class for both?

Reply:
Of course. We should avoid using tag name as a selector.

Reply:
OK, I added `.ez-bulk-operation-failed-modal__table` and `.ez-bulk-operation-failed-modal__table-body`.